src/docfx/restore/FileResolver.cs

```diff
@@ -178,14 +182,14 @@ private string DownloadFromUrlCore(string url)
             }
         }
 
-        private static string GetRestorePathFromUrl(string url)
+        private static PathString GetRestorePathFromUrl(string url)
         {
-            return PathUtility.NormalizeFile(Path.Combine(AppData.GetFileDownloadPath(url)));
+            return new PathString(PathUtility.NormalizeFile(Path.Combine(AppData.GetFileDownloadPath(url))));
```

Comment:
`new PathString` does the path normalization.
```suggestion
            return new PathString(Path.Combine(AppData.GetFileDownloadPath(url))));
```